generate-docs/script-inputs/local-repo-snippets.yaml

```diff
@@ -2457,7 +2457,7 @@ Office.AddinCommands.Event.completed:
         // Check if subject contains "BLOCK".
         outgoingMsg.subject.getAsync(function (result) {
             // Subject is in `result.value`.
-            // If value is found then stop send, else allow send.
+            // If search term is found then stop send, else allow send.

```

Comment:
Do we want to make this more "prose" sounding? Like "If the search term is found, then stop the message from being sent. Otherwise, allow the send." Or is that too high-brow for a code comment?

Reply:
Thanks for the feedback. I'll go ahead and check it in as-is but I'll consider refining that statement in a subsequent PR.

Reply:
Ended up with `// If search term "BLOCK" is found, don't send message.`